common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAuthorizationRequest.java

```diff
@@ -62,6 +63,7 @@
     public static final String LIB_ID_CPU = "x-client-CPU";
     public static final String LIB_ID_OS_VER = "x-client-OS";
     public static final String LIB_ID_DM = "x-client-DM";
+    private static final long serialVersionUID = 6873634931996113294L;
```

Comment:
please use all upper case letters for constants.

Reply:
This value should be left as-is: [it is used by the Android Runtime to compute when the classes change between versions](https://developer.android.com/reference/java/io/Serializable). Its value is inspected through reflection.